main/settings/src/main/scala/sbt/Reference.scala

```diff
@@ -78,7 +78,7 @@ object Reference {
       case LocalRootProject    => "{<this>}<root>"
       case LocalProject(id)    => "{<this>}" + id
       case RootProject(uri)    => "{" + uri + " }<root>"
-      case ProjectRef(uri, id) => "{" + uri + "}" + id
+      case ProjectRef(uri, id) => s"""ProjectRef(uri("$uri"),"$id")"""
```

Comment:
> In general, I think we are in agreement that we should try to use the Scala identifier for the references (e.g. root) where possible.
  
Can we remove the URI from the display function? I think it does not give any valuable information for most of the sbt users. It's verbose, especially for large paths (common case for most of devs). The idea would be to use only the project references (e.g. `root`).
  
I think that it's unlikely that people want this information. If they want it, they can either do `project` (which will show the uri) or `projectInfo` that will show the information of all the projects.
  
In my opinion, we should stick with what's common. And in that case that is references to project definitions. Note that this change should also be done for `{this}` and use `ThisBuild` instead. WDYT?